include/sound/soc-topology.h

```diff
@@ -34,6 +34,9 @@ struct snd_soc_dapm_route;
 /* object scan be loaded and unloaded in groups with identfying indexes */
 #define SND_SOC_TPLG_INDEX_ALL	0	/* ID that matches all FW objects */
 
+#define SOC_VIRT_DAI_PLAYBACK "VM FE Playback"
+#define SOC_VIRT_DAI_CAPTURE "VM FE Capture"
+
```

Comment:
you want this, and uses of these defines in a separate patch. It's not possible to upstream this otherwise.